DocumentsFromSnapshotMigration/src/test/java/com/rfs/FullTest.java

```diff
@@ -308,6 +314,90 @@ private DocumentsRunner.CompletionStatus migrateDocumentsWithOneWorker(SourceRep
         }
     }
 
+    public static Stream<Arguments> makeProcessExitArgs() {
+        var sourceImageArgs = SOURCE_IMAGES.stream().map(name -> makeParamsForBase(name)).collect(Collectors.toList());
+        var targetImageNames = TARGET_IMAGES.stream().map(SearchClusterContainer.Version::getImageName).collect(Collectors.toList());
+
+        return sourceImageArgs.stream()
+            .flatMap(a->
+            targetImageNames.stream().map(b->Arguments.of(a[0], a[1], a[2], b)));
+    }
+
+    @ParameterizedTest
```

Comment:
Why do you think this test needs to be parameterized on source/target endpoints?  I would rather see a parameter on something like shouldThrow vs shouldSucceed.

Reply:
I don't; I was just following the existing paradigm until I knew how to insert exceptions, which is what I really wanted to parameterize on.  I had to have something in the draft PR though.